scania-digital-design-system/sdds-website
# fix(icons): CORS issue

**Describe pull-request**  
CORS issue with downloading SVG icon/s in Chrome from AWS

**How to test**  
1. Open Amplify link down below
2. Go to Foundations -> Icons -> select "Icons" tab
3. Test 1: From gallery, click on one Icon and in modal after it click "Download", expected result is downloaded single icon. Find it in your files and try to open it. Icon is opened in full screen view.
4. Test 2: Froma gallery select couple icons using checkboxes, click button "Download selected", in modal click "Donwload". Selection is downloaded as zip file. Unzip it. Open one or more icons. They present in full screen view.
